query/security/ldap/src/main/java/org/codice/ddf/admin/ldap/discover/GetLdapConfigurations.java

```diff
@@ -44,7 +46,9 @@ public GetLdapConfigurations(ConfiguratorSuite configuratorSuite) {
 
   @Override
   public ListField<LdapConfigurationField> performFunction() {
-    return serviceCommons.getLdapConfigurations();
+    return AccessController.doPrivileged(
```

Comment:
❓ Should this be done in the `getLdapConfigurations` method?

Reply:
Our security policy takes care of that.